week1/assignment/styles.css

```
@@ -0,0 +1,161 @@
+@import "./reset.css";
+body {
+  background-color: black;  
+}
+header {
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: transparent;
+  padding: 1% 1.5%;
+  box-sizing: border-box; 
+}
+
+header a {
+  text-decoration: none;
+  margin-right: 5%;
+  font-size: 2rem;
+  color: white;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  width: 10%;
+  margin-right: 3%;
+}
+
+aside {
+  min-width: 144px;
+}
+
+aside i {
+  margin-right: 10%;
+  font-size: 2rem;
+}
+
+.homebanner-img {
+  width: 100%;
+  height: 80vh;
+  object-fit: cover;
+}
+
+.homebanner-text {
+  min-width: 300px;
+  position: absolute;
+  top: 60%;
+  right: 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.homebanner-text p {
+  padding: 8%;
+  background-color: white;
+  color: darkgreen;
+  font-weight: bold;
+  font-size: 3rem;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 1%;
+}
+
+.button {
+  text-decoration: none;
+  padding: 5%;
+  background-color: rgba(80, 80, 80, 0.5);
+  color: white;
+  border-radius: 10px;
+  font-size: 1.7rem;
+  transition: background-color 0.3s ease;
+}
+.button:hover {
+  background-color: rgba(0, 0, 0, 0.8);
+}
+
+.button i {
+  margin: 0 10px;
+}
+
+.gallery-title {
+  padding: 2%;
+  font-size: 2rem;
+  margin-top: 2%;
+  color: white;
+}
+
+.image-gallery {
+  display: flex;
+  overflow: hidden;
+  margin: 0 30px;
+}
+
+.image-gallery-wrapper {
+  display: flex;
+  width: max-content; 
+  animation: slide 30s linear infinite;
+}
+
+.image-gallery img {
+  width: 250px;
+  height: 250px;
+  object-fit: cover;
+  transition: transform 0.5s ease;
+  padding: 0 10px;
+}
+
+.image-gallery img:hover {
+  transform: scale(1.2);
+}
+
+@keyframes slide {
+  0% {
+    transform: translateX(0);
+  }
+  100% {
+    transform: translateX(-50%); 
+  }
+}
+
+.scroll-view {
+  display: flex;
+  width: 100%;
+  margin-bottom: 3%;
+}
+
+.scroll-button {
+  background: rgb(100, 100, 100);
+  color: white;
+  border: none;
+  cursor: pointer;
+  font-size: 24px;
+  padding: 10px;
+  display: flex;

```

p2: 여기는 `font-size`가 24px로 되어있네요! 2.4rem으로 고쳐주세요!!
```suggestion
  font-size: 2.4rem;
  padding: 1rem;
  display: flex;
```

---

네 수정하겠습니다.